movies/src/components/movieEditPopup/movieEditPopup.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import CloseButton from "../closeButton/CloseButton";
+import Modal from 'react-modal';
+import Button from "../common/button/Button";
+import Input from "../common/input/Input";
+import Dropdown from '../common/dropdown/Dropdown';
+import Datepicker from '../common/datepicker/Datepicker';
+import Constants from '../constants';
+import MovieActionPopup from "../movieActionPopup/MovieActionPopup";
+
+export default function MovieEditPopup (props) {
```

Comment:
Additionally, I'm unable to scroll inner modal content

Reply:
Fixed